client/components/spotlight/index.tsx

```diff
@@ -0,0 +1,72 @@
+import { Button } from '@automattic/components';
+import styled from '@emotion/styled';
+
+const SpotlightContainer = styled.div`
+	background-color: var( --studio-gray-0 );
+	display: flex;
+	justify-content: space-between;
+	align-items: center;
+	padding: 20px;
+`;
+
+const SpotlightContent = styled.div`
+	display: flex;
+	align-items: center;
+`;
+
+const SpotlightIllustration = styled.img`
+	height: 75px;
+`;
+
+const SpotlightTextContainer = styled.div`
+	margin-left: 20px;
+`;
+
+const SpotlightTitle = styled.div`
+	display: block;
+	color: var( --color-neutral-50 );
+	font-weight: 600;
+	font-size: 0.75rem;
+`;
+
+const SpotlightTagline = styled.div`
+	display: block;
+	color: var( --color-neutral-70 );
+	font-weight: 600;
+	font-size: 1rem;
+`;
+
+const SpotlightCta = styled.div`
+	max-height: 32px;
+`;
+interface SpotlightProps {
+	onClick?: () => void;
+	url: string;
+	taglineText: string;
+	illustrationSrc: string;
+	titleText: string;
+	ctaText: string;
+}
+
+const Spotlight: React.FunctionComponent< SpotlightProps > = ( props: SpotlightProps ) => {
```

Comment:
Since we are creating this new shiny component, I think it would worth also adding some track events for user who have clicked through it. This way we can measure the performance (click through rate) and run A/B tests later.

An example can be found here
https://github.com/Automattic/wp-calypso/blob/0178a4ae1aa2031071dc846cfc0344b353fd233e/client/blocks/announcement-modal/index.jsx#L52-L56

Reply:
Was thinking on leaving this responsibility to the parent component through the `onClick` but I think we can do it like this using a `spotlight_slug` or `spotlight_id` will add a follow-up for this one.